misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueConfig.kt

```diff
@@ -17,6 +17,12 @@ class AwsSqsJobQueueConfig(
    * External queues is a set of externally owned SQS queues accessed by this service, mapping
    * an internal queue name to the (account ID, region, name) of the queue in the external account
    */
-  val external_queues: Map<String, AwsSqsQueueConfig> = mapOf()
+  val external_queues: Map<String, AwsSqsQueueConfig> = mapOf(),
+
+  /**
+   * Number of consumer threads to run. This parameter controls the number of jobs handled by
+   * consumer concurrently.
+   */
+  val consumer_thread_pool_size: Int = 4
```

Comment:
Any suggestions for the comment and var name? :)

Reply:
the naming seems reasonable!

The second part of the comment doesn't seem quite accurate: "This parameter controls the number of jobs handled by consumer concurrently." Maybe like, "Number of threads to run for the job queue consumer"?

Reply:
number of jobs that can be processed concurrently

Reply:
Thanks for the feedback! I'll pick @ryanhall07 's idea `Number of jobs that can be processed concurrently.` if it's okay! :)